pkg/controller/.import-restrictions

```diff
@@ -101,13 +101,18 @@
         "github.com/google/gofuzz",
         "github.com/prometheus/client_golang/prometheus",
         "github.com/robfig/cron",
+        "github.com/spf13/afero",
         "github.com/spf13/pflag",
         "github.com/stretchr/testify/assert",
         "github.com/stretchr/testify/require",
         "github.com/google/gofuzz",
         "github.com/golang/protobuf/ptypes/wrappers",
         "github.com/golang/protobuf/proto",
-        "github.com/container-storage-interface/spec/lib/go/csi"
+        "github.com/container-storage-interface/spec/lib/go/csi",
```

Comment:
Why import restrictions in this package `pkg/controller/` being changed. I don't see changes in that package in this PR.

Reply:
`./pkg/controller/...` has a dependency on the `csi` pacakge:

```sh
 ~/workspace/kubernetes-playground/go/src/k8s.io/kubernetes/pkg/controller → go list -f '{{ join .Deps "\n" }}' ./... | grep csi | sort -u
k8s.io/kubernetes/pkg/volume/csi
k8s.io/kubernetes/pkg/volume/csi/csiv0
k8s.io/kubernetes/pkg/volume/csi/nodeinfomanager
k8s.io/kubernetes/vendor/github.com/container-storage-interface/spec/lib/go/csi
k8s.io/kubernetes/vendor/k8s.io/csi-translation-lib
k8s.io/kubernetes/vendor/k8s.io/csi-translation-lib/plugins
~/workspace/kubernetes-playground/go/src/k8s.io/kubernetes/pkg/controller → 
```

With this PR we introduce the dependency of to `csi` package to the `pluginwatcher` package, by [importing the `pluginwatcher` package directly](https://github.com/kubernetes/kubernetes/pull/73919/files#diff-f52821ec6236e86b02ca334533395c91R46).

It is actually the `pluginwatcher` package that has those dependencies, but via the `csi` package they became now a transitive dependency of `./pkg/controller/volume/...`.

```bash
~/workspace/kubernetes-playground/go/src/k8s.io/kubernetes/pkg/controller → go list -f '{{ join .Deps "\n" }}' ./... | grep pluginwatcher | sort -u
k8s.io/kubernetes/pkg/kubelet/util/pluginwatcher
k8s.io/kubernetes/pkg/kubelet/util/pluginwatcher/example_plugin_apis/v1beta1
k8s.io/kubernetes/pkg/kubelet/util/pluginwatcher/example_plugin_apis/v1beta2
~/workspace/kubernetes-playground/go/src/k8s.io/kubernetes/pkg/controller →
```

There was already a `.import-restrictions` for `./pkg/controller` but no more specific one for `./pkg/controller/volume`, so we decided to adapt the import restrictions which is responsible for all controllers, not only the volume controllers.